influxdb-backup/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM influxdb:1.2.4
+
+# Backup the following databases, seperator ":"
+ENV DATABASES=demo:_internal:testdb
```

Comment:
Need to use a query to find out the databases to back up. Must not put in this file because the user can dynamically create dbs.